### DISCUSSION TITLE
variable Zeitraumabfrage

### DISCUSSION THREAD

**Discussion by weinberg64** [solectrus/solectrus#3004] [Ideas]
es gibt viele statische Abfragen für diverse Zeiträume, aber keine Möglichkeit einen beliebigen Zeitraum z.B. von 01.02.YY?? bis 30.06.YY?? zu wählen. Die Zeitraumauswahl/Statistikanzeige könnte auf einer eigenen subsite dargestellt werden

**Comment by ledermann** [comment]
Ja, derzeitig können nur ganze Jahre, Monate, Wochen, oder Tage gewählt werden - und Gesamt. Ich stimme dir zu, dass es nützlich wäre, wenn man einen beliebigen Zeitraum auswerten könnte und habe #3007 angelegt. Edit: Alles Weitere in #1931
